langchain/callbacks/manager.py

```diff
@@ -571,6 +574,22 @@ def on_llm_new_token(
             **kwargs,
         )
 
+    def on_llm_retry(
+        self,
+        retry_state: RetryCallState,
```

Comment:
Can we make this arg something that is json serialisable? Analysing that python repl string wont be fun

Reply:
Ya I can take out the useful bits